app/lib/redcarpet/render/html_rouge.rb

```diff
@@ -14,10 +14,12 @@ def link(link, _title, content)
         @options[:link_attributes]&.each do |attribute, value|
           link_attributes += %( #{attribute}="#{value}")
         end
-        if (/\A(https?:\/\/)/.match? link) || link.nil?
+        if (/https?:\/\/[\S]+/.match? link) || link.nil?
           %(<a href="#{link}"#{link_attributes}>#{content}</a>)
-        else
+        elsif /\.{1}/.match? link
           %(<a href="//#{link}"#{link_attributes}>#{content}</a>)
+        else
+          %(<a href="https://dev.to#{link}"#{link_attributes}>#{content}</a>)
```

Comment:
Instead of hardcoding dev.to into the url maybe we could use the `APP_DOMAIN` environment variable so links are sane in whatever environment we are?

Reply:
Yes, I was looking for a source of truth for the current domain, thanks for pointing out to the env vars, I'll update it later today.

Reply:
Yeah, the `https` is technically often defined by the env. So...

```ruby
#{ApplicationConfig["APP_PROTOCOL"]}#{ApplicationConfig["APP_DOMAIN"]}
```